convert2rhel/actions/report.py

```diff
@@ -0,0 +1,128 @@
+# Copyright(C) 2023 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+__metaclass__ = type
+
+import logging
+import textwrap
+
+from convert2rhel import utils
+from convert2rhel.actions import find_actions_of_severity, format_report_message, format_report_section_heading
+from convert2rhel.logger import bcolors, colorize
```

Comment:
Remove import of bcolors: We're using a higher level function, colorize() instead.
```suggestion
from convert2rhel.logger import colorize
```